### PR TITLE
Inspect

### PR DESCRIPTION
**Pull request by LChernik**
feat: header, pages navigation, burger menu, "welcome" info section on the Home page, translation, started implementation of the pets gallery on the Home page

### CODE REVIEW

File: src/styles/media.ts [hunk 1]
```diff
@@ -34,7 +31,7 @@ export const media = {
 
 //     return acc;
 //   },
-//   {} as { [key in keyof typeof sizes]: MediaFunction },
+//   {} as { [key in keyof typeof sizes]: any },
```
Comment (kanstantsin-chernik) [inline src/styles/media.ts:34]: Remove garbage comments

File: src/app/components/Welcome/Data.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const homeObjOne = {
+    buttonLabel: 'view live cam',
+    headline: 'Watch your favorite animal online',
+    description: 'Explore the exciting and mysterious world of wild animals in a natural setting without leaving your home.',
```
Comment (kanstantsin-chernik) [inline src/app/components/Welcome/Data.tsx]: Shouldn't these be translated?

File: src/app/pages/ContactPage/index.tsx [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
-import { Header } from 'app/pages/HomePage/Header';
 
-export function HomePage() {
+export function ContactPage() {
+
   return (
     <>
       <Helmet>
-        <title>Home Page</title>
+        <title>Contact Us Page</title>
```
Comment (kanstantsin-chernik) [inline src/app/pages/ContactPage/index.tsx:9]: should this be translated as well as description?

**Review by kanstantsin-chernik — COMMENTED** [review]
In general PRs should be smaller, description should be broader, and it is good to document what you've done in Readme and code here and there

File: src/app/components/Welcome/Data.tsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export const homeObjOne = {
+    buttonLabel: 'view live cam',
```
Comment (kanstantsin-chernik) [inline src/app/components/Welcome/Data.tsx]: Should be translated

File: src/app/pages/MapPage/index.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet-async';
+
+export function MapPage() {
+
+  return (
+    <>
+      <Helmet>
+        <title>Map Page</title>
+        <meta name="description" content="A Boilerplate application homepage" />
```
Comment (kanstantsin-chernik) [inline src/app/pages/MapPage/index.tsx]: Change discription

File: src/app/pages/ZoosPage/index.tsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet-async';
+
+export function ZoosPage() {
+
+  return (
+    <>
+      <Helmet>
+        <title>Zoos Page</title>
+        <meta name="description" content="A Boilerplate application homepage" />
```
Comment (kanstantsin-chernik) [inline src/app/pages/ZoosPage/index.tsx]: Change discription

File: src/app/components/Welcome/Data.tsx (hunk 4 above)
Comment (LChernik) [reply src/app/components/Welcome/Data.tsx]: The file should be removed

File: src/app/components/Welcome/Data.tsx (hunk 2 above)
Comment (LChernik) [reply src/app/components/Welcome/Data.tsx]: The file should be removed

File: src/app/pages/ContactPage/index.tsx (hunk 3 above)
Comment (LChernik) [reply src/app/pages/ContactPage/index.tsx:9]: File title shouldn't be translated

File: src/styles/media.ts (hunk 1 above)
Comment (LChernik) [reply src/styles/media.ts:34]: I need the template code for now

**Review by kanstantsin-chernik — APPROVED** [review]
lgtm

**Pull request merged**